scripts/linters/python_linter.py

```diff
@@ -117,6 +117,52 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_non_test_files(self):
+        """This function is used to check that function
+           with test_only in their names are in test files.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting function defintion checks')
+            python_utils.PRINT('----------------------------------------')
+        summary_messages = []
+        files_to_check = self.py_filepaths
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+            for filepath in files_to_check:
+                if not filepath.endswith('_test.py'):
+                    for line_num, line in enumerate(FILE_CACHE.readlines(
+                            filepath)):
+                        line = line.strip()
+                        words = line.split()
+                        if len(words) < 2:
+                            continue
+                        ind1 = words[0].startswith('def')
+                        ind2 = words[1].startswith('test_only')
+                        if ind1 and ind2:
+                            summary_message = (
+                                'ERROR:   Non test file uses test functions at'
+                                ' line:', python_utils.UNICODE(line_num + 1),
+                                ' in file:', filepath
+                                )
+                            # To convert tuple into string.
+                            s = ''.join(summary_message)
```

Comment:
Use string formatting `%s`

Reply:
Format your message like this:
`filename --> Line line_num: error_message`

Reply:
done